recipes/stopit/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "stopit" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "d879a892e6c0d524763b8bf20831a78311d86ea59ec199ec551abf3f494b0708" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+
+
+test:
+  imports:
+    - stopit
+
+about:
+  home: https://pypi.python.org/pypi/stopit
+  license: GPL-3.0
+  license_family: GPL
```

Comment:
Could you submit an issue upstream asking them to include a license file in the source distribution and put a link to the issue as a comment here.

Reply:
[PR](https://github.com/glenfant/stopit/pull/15)!